examples/browser/package.json

```diff
@@ -32,7 +32,7 @@
     "@theia/vsx-registry": "latest",
     "@theia/keymaps": "latest",
     "@theia/getting-started": "latest",
-    "theia-trace-viewer": "0.0.0"
+    "@trace-viewer/theia-trace-extension": "0.0.1"
```

Comment:
I do not think this scope this will work, to publish under the `theia` npm user

"Each npm user/organization has their own scope, and only you can add packages in your scope."
https://docs.npmjs.com/cli/v7/using-npm/scope

So, I think you can either create your own npm "trace-viewer" user/scope and publish under that user, use the `@theia` scope or use no scope and publish under the `theia` npm user.

Reply:
(assuming this is a package you intend to publish - I did not look in-depth)

Reply:
> (assuming this is a package you intend to publish - I did not look in-depth)

Thanks Marc for the information. We'd like to publish the 3 following packages:

-     \@trace-viewer/base
-     \@trace-viewer/react-components
-     \@trace-viewer/theia-trace-extension

We used \@trace-viewer to indicate that they below together. But I don't actually want to create that scope. So I guess I remove the scope an rename the packages "base" and "react-components" to something clearer.

Reply:
Yes, that would work. Even smt like: 
`traceviewer-base`
`traceviewer-react-components`
`traceviewer-trace-extension` (is it really an extension? if a Theia extension, you could use e.g. `@theia/trace-viewer-extension` - it's allowed to mix and match, packages with and without a scope)

Reply:
Your suggestions are good proposals. However, removing the scope from @trace-viewer/base and @trace-viewer/react-components many source code files need to be changed to use new names. Not clear yet the way forward I should take. I'll need some more thinking.

Reply:
I think the following name suitable:

`traceviewer-base`
`traceviewer-react-components`
`theia-traceviewer`

Reply:
> theia-traceviewer

is this a pure JS library or a Theia extension?  If the later, I would go with `@theia/traceviewer`.

Reply:
Your name will work too, so at the end of the day, it's your decision.